\begin{abstract}

This study employed atomistic molecular dynamics (MD) simulations to examine the binding interactions between amyloid fibrils (A$\beta$(1––42) and tau) with various ligands: anionic pFTAA, qFTAA-CN, neutral HS-276, and cationic bTVBT4. The ligand––fibril interactions were analyzed in a two-step approach. First, a qualitative analysis of the spatial distribution of the ligands around amyloid fibril protofilaments, as visualized through density plots, identified binding sites and their population density. Second, a quantitative assessment of the binding energy for each ligand––fibril pair was conducted, employing umbrella sampling. The results show that pFTAA and qFTAA-CN share a common binding site in both A$\beta$(1––42) and tau fibrils. On the other hand, bTVBT4 and HS-276 each have distinct binding sites in A$\beta$(1––42), but they share the same site in tau fibrils. Moreover, a comparative analysis of each ligand—fibril pair reveals the lowest population densities and binding energies occur when bTVBT4 interacts with A$\beta$(1––42) and HS-276 with the tau fibril. These findings are corroborated by experiments in which these ligand––fibril pairs exhibit no fluorescence, thereby validating the simulation results. Building upon these findings, a detailed structural analysis was conducted to understand how ligand conformations and interactions vary between the protein-fibril and aqueous environments. Furthermore, the binding of anionic ligands to fibrils is driven by Coulombic forces, whereas for neutral or lower magnitude positive charge ligands, it is driven by Lennard-Jones forces. This highlights the influence of charge magnitude on binding, providing insights for the design of future ligands targeting A$\beta$(1––42) and tau fibrils.


\end{abstract}

\clearpage

\section*{Introduction}

Neurodegenerative diseases such as Alzheimer's, Parkinson's and Pick's are associated with the aggregation of misfolded proteins where normally structured proteins lose their typical shape and form aggregates.\cite{Soto2018Oct,Gandhi2019May,Saijo2017May} Particularly in Alzheimer's disease (AD), the misfolded proteins amyloid-beta (A$\beta$(1––42)) and tau accumulate as deposits in the brain.\cite{Ross2004} The pathogenesis of AD involves the development of A$\beta$(1––42) fibrils in the neocortex, which then spread into deeper brain regions.\cite{Thal2002Jun,grothe2017} In contrast, tau fibril deposits originate in the limbic region and gradually extend outward towards the neocortex.\cite{Arnold1991,Braak1991,D'Errico2020Aug} Recent literature suggests that such propagation is associated with a transition from the pre-clinical to the clinical phase.\cite{Pontecorvo2017,Wang2016} Therefore, the early detection of A$\beta$(1––42) and tau is of paramount importance, as it could potentially pave the way for timely interventions that might alter the course of the disease.



One promising approach for detecting these pathological lesions involves the use of fluorescent ligands, which are molecules capable of binding to AD-related lesions and emitting detectable light in the visible region of the electromagnetic spectrum. These ligands can be further converted into positron emission tomography (PET) tracers by labeling one of the atoms within the ligand, thereby enabling real-time monitoring of AD pathogenesis.\cite{Nordeman2016} The selection of appropriate fluorescent ligands is critical, as these ligands must possess specific characteristics to effectively detect AD-related lesions.\cite{Nunez-Diaz2024Jan} First and foremost, these ligands should have the ability to penetrate the blood-brain barrier to ensure they can reach the target areas within the brain. Additionally, they should exhibit environmental sensitivity, meaning their optophysical properties change in response to different environments. These spectral changes can be attributed to both molecular and electronic structural alterations. 

To elucidate the molecular structural changes that underlie the desired environmental sensitivity and spectral shifts, molecular dynamics (MD) simulations play a crucial role. By simulating the interactions between ligands and amyloid fibrils in different environments, such as water or within proteins, and calculating the dihedral distributions of important rotatable bonds, insights into the structural changes in varying environments can be gained. Furthermore, MD simulations can uncover the binding modes of ligands to protein fibrils and the energy landscapes of these complexes. Comparative studies among multiple ligands can provide insights into the factors driving fibril binding, enabling the design of ligands that selectively bind to chosen sites while avoiding others. Elucidating specific binding sites is advantageous, as it facilitates the development of ligands tailored to target desired regions while avoiding unintended interactions. For instance, a new generation of ligands could be designed to avoid the binding sites common to the U.S. Food and Drug Administration (FDA) approved ligand [18F]AV-1451, which has been shown to produce misleading signals due to off-target binding.\cite{Vermeiren2018}




\begin{figure}[ht]
	\centering
	\includegraphics[width=0.7\textwidth]{Figures/manuscript/ligands_v4.pdf}
	\caption{Molecular structures of the anionic ligand (p-FTAA and q-FTAA-CN), neutral ligand (HS-276), and catioic ligand  (bTVBT4) . 
             }
	\label{fig:ligands}
\end{figure}

Ligands based on the thiophene moiety have shown promising results for binding to amyloid fibrils.  Molecular structures of these ligands are presented in Figure \ref{fig:ligands}. Combining the thiophene moiety with other structural motifs facilitates targeted protein binding. For example, the bTVBT4 ligand, which consists of a bi-thiophene unit combined with benzothiazole, binds exclusively to tau aggregates.\cite{Shirani2017} The HS-276 ligand, comprising a bi-thiophene unit combined with an azaindole moiety, demonstrates preferential binding towards A$\beta$(1––42) aggregates.\cite{Klingstedt2021} The pFTAA, a luminescent conjugated poly-thiophene (LCP), binds to both hallmark tau and A$\beta$(1––42) folds of AD.\cite{Aslund2009} In contrast, qFTAA-CN, another LCP ligand, binds to A$\beta$(1––42).\cite{Fandrich2018} However, its binding to tau aggregates remains ambiguous. Despite all four ligands sharing a thiophene moiety, they display diverse binding functionalities towards amyloids.

In the current study, we employ a computational approach to elucidate the differential binding affinities of a diverse array of ligands, namely pFTAA, qFTAA-CN, HS-276, and bTVBT4, when in complex with the proteins A$\beta$(1––42) and tau fibrils. These simulations are analyzed through a two-step approach to provide a comprehensive comparative picture. First, we assess the distribution of ligands around the protofilament of each fibril qualitatively by examining their spatial distribution. Secondly, we perform a quantitative evaluation of the binding free energy. Furthermore, we compare the structural characteristics of each ligand, as observed in the ligand–-fibril complex, to those in a water environment. The results suggest that the binding of ligands is driven by Coulombic interactions for anionic ligands carrying a high magnitude of charge. In contrast, for neutral and cationic ligands with a single positive charge, it is driven by Lennard––Jones (LJ) interactions. Additionally, it is noteworthy that the characteristics of these ligands in the fibril complex significantly differ from their behaviour in an aqueous environment. This study therefore offers valuable insights into the molecular mechanisms governing ligand–fibril interactions.

\section*{Computational details}

The binding interactions between A$\beta$(1––42) and tau fibrils with ligands pFTAA, qFTAA-CN, HS-276, and bTVBT4 were investigated using atomistic molecular dynamics (MD) simulations. Comprehensive details of the computational methodology are provided in the Supporting Information (SI). The starting structures for A$\beta$(1––42) and tau fibrils were based on cryo-EM structures and obtained from earlier research.\cite{Konig2018, todarwal_2021_tau} The initial parameters for qFTAA-CN, pFTAA, and HS-276 ligands, sourced from the General Amber Force Field (GAFF),\cite{GAFFa,GAFFb} were further refined and optimized using quantum chemical calculations. This refinement process included the use of the Gaussian program for obtaining optimized geometries and dihedral scanning,\cite{g16,gv6} and the Dalton program for performing single-point absorption spectrum calculations,\cite{dalton2014} as elaborated in Section 1 of the SI. Parameters for bTVBT4 were adopted from prior studies.\cite{todarwal_2021_tau} Amyloid fibrils consisting of amino acid residues were modeled using the standard Amber force field parameters,\cite{Maier2015Aug} and used the TIP3P force field for water molecules.\cite{Jorgensen1983}

SI Section 2.2 elaborates on the MD simulations performed to identify potential binding sites on A$\beta$(1––42) and tau protein fibrils for each ligand. These calculations utilized the full periodic model of either A$\beta$(1––42) or tau fibrils, combined with 60 ligand molecules, comprising 15 molecules each of pFTAA, qFTAA-CN, HS-276, and bTVBT4. MD simulation trajectories were analyzed using the spatial distribution function tool in VIAMD package\cite{VIAMD,Skanberg2023Dec}, resulting in ligand density plots around the amyloid fibrils. These plots, were verified to have achieved convergence over the simulation time (see SI Figure S5 for details).

For a more in-depth examination of the binding modes of each ligand at the identified sites, a smaller protein fibril model system was extracted from the full periodic model, as described in SI Section 2.3. This reduced model comprises ten filaments of either A$\beta$(1––42) or tau fibrils, each docked with one of the four ligands. To maintain the distance between amyloid filaments in the reduced model, position constraints were applied to the outermost protein filament during the MD simulations. From these smaller model simulations, strong binding modes for each ligand were identified. Starting from these identified strong binding modes, the free energy profile was determined using Potential of Mean Force (PMF) calculations, employing umbrella sampling method. A series of windows were generated along the reaction coordinate, defined as the distance between the center of mass of residues that form a binding site on a fibril and a ligand. Harmonic restraints were applied at each window to effectively sample the behavior of the system. Upon completing the MD simulations for each window, the Weighted Histogram Analysis Method (WHAM) was employed to compile the data and construct the PMF curve.\cite{Hub2010Dec} This curve reveals the free energy landscape of the ligand––fibril interaction. Further details regarding PMF calculations for various binding sites are found in SI Section 2.4.

Finally, simulations of pFTAA, qFTAA-CN, and HS-276 in a water solvent were also conducted to compare these results with those obtained in the protein environment. Details of these simulations are provided in SI Section 2.1. For bTVBT4 in water, simulation data were sourced from previous work.\cite{todarwal_2021_tau}


\section*{Results and Discussion}

In previous studies, unbiased MD simulations were used to identify the binding sites of anionic pentameric oligothiophene (pFTAA) and bi-thiophene-benzothiazole (bTVBT4) when interacting with A$\beta$(1––42) and tau fibrils, respectively. In this article, we use a similar MD methodology to compare the binding properties of four ligands (pFTAA, qFTAA-CN, HS-276, and bTVBT4) to amyloid fibrils (A$\beta$(1––42) and tau). The structured cores of A$\beta$(1––42) and tau are potential targets for the ligands, as disordered peptides attached to these cores are expected to dissociate\cite{Zhang2010,fitzpatrick_cryo-em_2017}. Therefore, we adopted the cryo-EM structure-based A$\beta$(1––42) and tau fibrils from our previous studies as a starting point for MD simulations. The A$\beta$(1––42) and tau fibrils were simulated independently with an equal mixture of four ligands (i.e., 15 molecules of each ligand type). This artificially high concentration of the ligands was used to quickly explore the protein––ligand energy landscape. However, for the analysis of spatial distribution, only ligand molecules that bound to the fibrils as single units were considered. Through spatial distribution analysis, the density of each ligand around the protein is comprehensively characterized, thereby pinpointing the specific binding sites. By leveraging this visualization technique, we can gain valuable insights into the protein––ligand interactions and identify key residues that contribute to the formation of stable complexes for further analysis.

The following sections provide a detailed examination of ligand interactions with A$\beta$(1––42) and tau fibrils, as well as the behavior of these ligands in a water environment. Initially, the binding properties of ligands to the A$\beta$(1––42) fibril are analyzed, and  a comparative analysis of their interactions is conducted. Subsequently, we will delve into a similar examination of ligand interactions with tau fibrils. In both sections, we also discuss possible unique aspects of ligand behavior in a water environment, differentiating it from their behavior within protein environments. Ultimately, we conclude by reflecting on the general binding patterns exhibited by these ligands.   

\subsection{Comparative Analysis of Ligand Interactions with A$\beta$(1––42) Fibril}

The spatial distribution analysis reveals distinct binding sites for each ligand type (anionic, cationic, or neutral) on the A$\beta$(1––42) fibril (see Figure \ref{fig:Density plot}(a)). Interestingly, the anionic ligands pFTAA and qFTAA-CN share the same strongest binding site (S1), which is located around amino acid residues LYS16, VAL18, and PHE20. The presence of LYS16, which is positively charged, likely attracts these negatively charged anionic ligands, contributing to the formation of a stable binding interaction. The neutral ligand HS-276 binds to two sites on the A$\beta$(1––42) fibril, with the strongest binding site (S2) involving residues ASP7, SER8, GLY9, and TYR10, which forms a pocket where HS-276 appears to fit perfectly. The secondary binding site for HS-276 is near residues PHE20 and ALA21, which overlaps with the binding site for anionic ligands. The cationic ligand bTVBT4 exhibits a distinct binding site (S3) comprising residues TYR10, GLU11, and VAL12, which are located near site S2. At site S3, bTVBT4 interacts strongly with TYR10 due to both Coulombic and LJ interactions, while the interaction with negatively charged GLU11 is weak because atoms with partial negative charges are away from the ligand. Since it only interacts mainly with one residue, TYR10, bTVBT4 showcases weak binding. All ligands bind to their respective sites parallel to the A$\beta$(1––42) fibril axis. The binding sites S1 and S3 have been previously reported in the literature for other ligands.\cite{Murugan2018Jul} However, we do not find any mention of site S2 in the existing literature where the HS-276 ligand binds. 

Conformational preferences of ligands within the A$\beta$(1––42) environment and aqueous solution exhibit substantial differences, which can be attributed to varying electrostatic and van der Waals interactions, among other factors. For instance, pFTAA and qFTAA-CN predominantly adopt TT (94\%) and TTT (70\%) conformations, respectively, when bound to proteins, while in water, they populate multiple 
conformations (see Figure \ref{fig: Ligand dihedral distribution}).   Their binding to the A$\beta$(1––42) protein is driven by Coulombic interactions, as seen in Table \ref{tab:interaction_energies}. When pFTAA or qFTAA-CN are in their respective TT and TTT conformations, their carboxyl groups are aligned in the same direction, facilitating stable protein––ligand complex formation through strong Coulombic interactions with the positively charged LYS16 residues at site S1. This directional alignment of carboxyl groups is critical for the binding of these anionic ligands. In contrast, HS-276 demonstrates more dynamic behavior in both environments by populating multiple conformations, with a slightly higher population of the Tt (44\%) conformation. Notably, bTVBT4 displays a significant increase in the C (79\%) conformation population within the A$\beta$(1––42) environment. The binding of the neutral ligand HS-276 and cationic bTVBT4 to the A$\beta$(1––42) protein is mainly driven by LJ interactions (see Table \ref{tab:interaction_energies}). HS-276 binds to site S2 and involves significant interactions with the TYR10 and GLY9 residues. For bTVBT4, its binding to site S3 of the A$\beta$(1––42) protein fibril primarily involves TYR10 through $\pi-\pi$ stacking. However, long simulations of the smaller bTVBT4–A$\beta$(1––42) model system indicate that this binding is not strong and eventually dissociates from the site.

The binding energy profiles of the ligands with A$\beta$(1––42) are illustrated in Figure \ref{fig:PMFs}(a). Both anionic ligands exhibit similar profiles and begin to interact with the fibril when they are within 2 nm of the binding site region. At approximately 0.5 nm from this region, there is an intermediate state where only one carboxyl group side of the ligand interacts with LYS16. For HS-276, the PMF profile indicates that it starts to strongly interact with its binding site from approximately 1.0 nm distance, while bTVBT4 starts interacting with its binding site at about 1.5 nm from its position. The results also reveal that pFTAA has the highest binding energy (11.5 kcal/mol), followed by qFTAA-CN (10.8 kcal/mol) and HS-276 (10.5 kcal/mol). On the other hand, bTVBT4 displays the lowest binding energy (7.94 kcal/mol). These findings are consistent with previous experimental studies demonstrating the potential of pFTAA, qFTAA-CN, and HS-276 as reliable amyloid-binding probes and confirming their binding to A$\beta$(1––42) fibrils. The low binding energy of bTVBT4 aligns with experimental observations that this ligand does not exhibit fluorescence, suggesting it may not bind to A$\beta$(1––42) fibrils and might not be suitable for diagnosis compared to the other three ligands. Interestingly, BTA-3, a neutral ligand based on the benzothiazole moiety combined with two cyanide groups, has been reported to bind strongly at the surface site S1 of  A$\beta$(1––42).\cite{ArulMurugan2018Aug} In contrast, bTVBT4, which also contains a benzothiazole moiety, shows decreased affinity. This suggests that its combination with bi-thiophene reduces binding to A$\beta$(1––42). Moreover, BTV-3 and qFTAA-CN possess cyanide groups, which might be helping them bind strongly at the same site S1.

When examining the spatial distribution of ligands around the A$\beta$(1––42) protofilament, it is evident from Figure \ref{fig:Density plot}(a) that there are significant differences in population density among the ligands. Specifically, pFTAA, qFTAA-CN, and HS-276 show a higher population densities compared to bTVBT4. This disparity in density can be primarily attributed to the relative affinities of these ligands toward A$\beta$(1––42). In particular, bTVBT4 exhibits a lower population density, which may be a direct consequence of its reduced affinity towards A$\beta$(1––42).

\begin{figure}[!htbp]
     \includegraphics[width=0.9\textwidth]{Figures/manuscript/Density_plot_3.pdf}
	\caption{The spatial distribution of ligands around an (a)A$\beta$(1––42) (Parameters: Density scaling = 9.5 and Alpha scaling 1.0) and (b) tau proto-filaments (Parameters: Density scaling = 5.0 and Alpha scaling 1.0). The plot is drawn over the last 100 ns simulation data from two independent simulations of a system consisting 60 ligands (15 molecules of each ligand type) with A$\beta$(1––42) fibril or tau fibril. Only ligands that bind to a fibril as a monomer were considered for calculating density. Additionally, calculated binding energies are given for each ligand––fibril combination (For bTVBT4––tau, the binding energy was adopted from previous work \cite{todarwal_2021_tau}. }
	\label{fig:Density plot}
\end{figure}

\begin{figure}[!htbp]
    \centering
    \includegraphics[width=\textwidth]{Figures/manuscript/PMF_aB_tau.pdf}
    \caption{
    The potential mean force (PMF) profiles obtained from multiple pulling trajectories for each of the four ligands when interacting with the (a) $A\beta(1-42)$ and (b) tau fibrils. The PMF profiles were generated by pulling the ligands from their most stable conformer at their respective binding pockets. The PMF profile reflects the free energy landscape of the ligand––fibril interaction and provides insight into the stability of the complex at different stages of the pulling process. In addition, it also reveal differences in the strength and nature of their interactions with the fibrils.}
    \label{fig:PMFs}
\end{figure}

\begin{figure}[!htbp]
     \includegraphics[width=0.7\textwidth]{Figures/manuscript/dihedral_distribution_3.pdf}
	\caption{ Percentage of relevant conformers of ligands based on important dihedrals (labeled in corresponding molecular structure) in an environment of water solvent, A$\beta$(1––42), and tau. The dihedrals of type SCCS are marked with "T" or "C" and of type SCCN is marked with "t" or "c". The statistics of ligands in a protein environment are determined from small binding models. }
	\label{fig: Ligand dihedral distribution}
\end{figure}


\begin{table}[!ht]
\caption{\textmd{Lennard––Jones (LJ) and Coulombic interaction energies (in kcal/mol) between ligands and amyloid fibrils ($A\beta$(1-42)  or tau). Data extracted from a 500 ns (except for the bTVBT4––A$\beta$) MD simulation of a small amyloid fibril binding model. For calculation details see SI Section 2.3 }} 
\label{tab:interaction_energies} 
\begin{tabular*}{\textwidth}{@{\extracolsep{\fill}} l ccccc } 
\hline
\hline
Fibril & Type & pFTAA & qFTAA-CN & HS-276 & bTVBT4 \\ 
\hline
\rule{0pt}{14pt}


$A\beta$42  & LJ      & $-26 \pm 4$    & $-24\pm 4$   & $-26\pm 2$ & $-24\pm 4$  \\
          & Coulomb & $-213\pm 33$  & $-159\pm 35$  & $-5\pm 6$    & $-16\pm 5$   \\
\cline{3-6}
\rule{0pt}{14pt}

tau  & LJ      & $-28\pm 5$ & $-31\pm 4$ & $-26\pm 2$ & $-24\pm 2$  \\
     & Coulomb & $-194\pm 32$ & $-184\pm 24$ & $-2\pm 2$ & $-7\pm 3$  \\
\hline
\hline
\end{tabular*}
\end{table}

\clearpage

\subsection{Comparative Analysis of Ligand Interactions with Tau Fibril}

The spatial distribution plots illustrated in Figure \ref{fig:Density plot}(b) display the interactions between the ligands and tau fibril, highlighting strong binding sites for pFTAA, qFTAA-CN, HS-276, and bTVBT4. Among these ligands, pFTAA and qFTAA-CN exhibit the strongest binding at site S1, where they primarily interact with the amino acid residues ILE360, HIS362, PRO364, GLY367, and LYS369. At this site, analogous to their binding with A$\beta$(1––42), these anionic ligands display a strong affinity for the positively charged LYS369. Additionally, they demonstrate weak interactions at two other sites, as depicted in Figure \ref{fig:Density plot}(b).  The positively charged ligand bTVBT4 and the neutral ligand HS-276 bind to site S2, interacting with the amino acid residues ILE360, THR361, and HIS362. However, bTVBT4 exhibits a stronger interaction with ILE360 and HIS362, while HS-276 primarily interacts with ILE360 and shows a weaker interaction with HIS362 due to a reduced Coulombic interaction. Interestingly, there is one minor binding site common to all four ligands (see Figure \ref{fig:Density plot}(b)), which has been reported in various experimental and theoretical studies, suggesting that it is easily accessible to most ligands.\cite{Murugan2018Jul,Shi2021May,Murugan2021Jul}. Complementing these binding site observations,  the interaction energies given in Table \ref{tab:interaction_energies} show that the binding of pFTAA and qFTAA-CN to the tau fibril is driven by Coulomb interactions, while the binding of HS-276 and bTVBT4 is driven by LJ interactions. 

The analysis of conformer distribution for ligands, when interacting with their high-affinity binding sites and comparing their distribution in water, demonstrated unique patterns. Specifically, pFTAA and qFTAA-CN at site S1 of tau, were primarily found in TT (75\%) and TTT (96\%) conformers, respectively (see Figure \ref{fig: Ligand dihedral distribution}). In contrast, in a water environment, pFTAA and qFTAA-CN were present only at 18\% and 19\%, respectively. As mentioned earlier, this structural alignment allows the carboxyl groups in pFTAA and qFTAA-CN to form strong Coulombic interactions with LYS369. For HS-276 and bTVBT4 at site S2, they favored Tt and T conformers, respectively. Intriguingly, the protein environment induced significant alterations in the conformer patterns of all four ligands when compared to their behavior in water. Furthermore, this shift in conformers is primarily influenced by the dihedral type, SCCS, which displayed a strong preference for the trans position in all ligands within the tau environment. These observations imply that ligand binding can trigger structural changes that impact the conformer distribution, which may, in turn, potentially alter the photophysical properties of the ligands.

As a result of the observed conformer distribution patterns, the binding energies of the ligands provide further insight into the relationship between their structural preferences and interaction with protein binding sites. It has been found that pFTAA exhibits the highest binding energy of 29.7 kcal/mol at site S1, followed closely by qFTAA-CN with a binding energy of 21.7 kcal/mol at the same site. At site S2, bTVBT4 shows a binding energy of 7.9 kcal/mol, while HS-276 has a slightly lower binding energy of 7.1 kcal/mol. Notably, the binding PMF profiles of pFTAA and qFTAA-CN are remarkably similar, suggesting a barrierless binding process to site S1, with interactions commencing from a considerable distance. In contrast, bTVBT4 and HS-276 exhibit comparable binding profiles, and although a barrierless path is available, their interaction with site S2 only commences from a shorter distance.

Experimental results align with the calculated relative binding energies of the ligands, with pFTAA and bTVBT4 displaying fluorescence upon binding to tau fibrils, whereas HS-276 does not. The difference in binding energy between bTVBT4 (which fluoresces) and HS-276 (which does not) is only 0.8 kcal/mol, falling within the error range of the PMF method used to calculate free energy. The ambiguous fluorescence behavior of qFTAA-CN observed in some experiments might be attributed to post-translational modifications of tau, specifically the acetylation of LYS369 at site S3, which could weaken its binding to tau and inhibit fluorescence.\cite{Alquezar2021Jan} Further investigation is required to confirm this hypothesis and to explore whether pFTAA can bind to site S3 even after post-translational modifications, especially given its higher binding energy compared to qFTAA-CN.

In the analysis of ligand distribution surrounding the tau protofilament, as depicted in Figure \ref{fig:Density plot}(b), a distinct pattern emerges. Unlike the A$\beta$(1––42) case, here,  HS-276 is characterized by a lower population density around the tau protofilament, whereas pFTAA, qFTAA-CN, and bTVBT4 exhibit a comparatively high density. This difference in density likely stems from the differential affinity of HS-276 towards the tau fibrils.



\section*{conclusions}
This study employed MD simulations, spatial distribution function (SDF), and umbrella sampling methods to investigate the comparative binding properties of four ligands (pFTAA, qFTAA-CN, HS-276, and bTVBT4) to amyloid fibrils (A$\beta$(1––42) and tau). The results underscore the crucial role of ligand charge magnitude and structural characteristics in shaping the specificity of ligand––fibril interactions. These findings shed light on the primary factors governing ligand binding to fibrils, elucidating whether the binding is predominantly driven by Coulombic or LJ interactions. Furthermore, the study illustrates how the fibril environment influences the conformational preferences of the ligands, leading to significant alterations in conformer patterns compared to their behavior in a water environment. The binding energy calculations provide valuable insights into the relationship between the structural preferences of ligands and their interactions with protein binding sites. The alignment of these findings with experimental results further supports their validity. Additionally, the spatial distribution analysis could serve to provide a qualitative picture of ligand-fibril interactions, without incurring additional computational cost. This work not only advances our understanding of the molecular mechanisms governing ligand––fibril interactions but also provides a foundation for the development of novel ligands as potential diagnostic tools in neurodegenerative diseases such as Alzheimer's and other amyloid-related disorders.


---
Above is the manuscript